firewood/src/merkle.rs

```diff
@@ -1190,6 +1190,79 @@ impl<S: ShaleStore<Node> + Send + Sync> Merkle<S> {
             merkle: self,
         })
     }
+
+    pub(crate) async fn range_proof<K: api::KeyType + Send + Sync>(
```

Comment:
crate->super; this method is only needed by parents of merkle, not anywhere in the crate.